css/index.css

```
@@ -112,7 +123,20 @@ h1, h2, h3, h4, h5 {
     .nav a {
         text-decoration: none;
         color: lightgray;
-        margin: 18px;
+        padding: 18px;
```

```suggestion
        padding: 18px 3.6%
```
coming from 18px  / 500px